server/athenian/api/openapi/openapi.yaml

```
@@ -2847,6 +2847,11 @@ components:
           items:
             $ref: '#/components/schemas/PullRequestProperty'
           type: array
+        stages:
```

for consistency with the returned PRs, I'd name this `stages_time_machine`
```suggestion
        stages_time_machine:
```

---

Nah, this suggests that `stages_now` can exist in the request, which is false. Too complex.

---

Why does it suggest such a thing? the request does not contain that field.
My proposal just make explicit the meaning of the stage used to filter the PRs